UefiTestingPkg/FunctionalSystemTests/MpManagement/Driver/AARCH64/SuspendHandling.c

```diff
@@ -586,6 +586,9 @@ CpuArchSleep (
     goto Done;
   }
 
+  ArmDisableMmu ();

```

Comment:
Does this apply to all power states for this function, beyond C3?

Reply:
Correct, CpuArchSleep is for C3 handling only, the others states will be issued through a separate routine (clock gate, etc.)